runtime/integration-tests/src/liquidity_pools/pallet/development/test_net.rs

```diff
@@ -114,6 +117,12 @@ pub fn para_ext(parachain_id: u32) -> sp_io::TestExternalities {
 		.balances(vec![
 			(AccountId::from(ALICE), CurrencyId::Native, cfg(10_000)),
 			(AccountId::from(BOB), CurrencyId::Native, cfg(10_000)),
+			(AccountId::from(ALICE), AUSD_CURRENCY_ID, AUSD_ED),
+			(AccountId::from(BOB), AUSD_CURRENCY_ID, AUSD_ED),
+			(AccountId::from(ALICE), USDT_CURRENCY_ID, USDT_ED),
+			(AccountId::from(BOB), USDT_CURRENCY_ID, USDT_ED),
+			(AccountId::from(ALICE), GLMR_CURRENCY_ID, GLMR_ED),
+			(AccountId::from(BOB), GLMR_CURRENCY_ID, GLMR_ED),
```

Comment:
Initial state with `ED` for all currencies in accounts used if foreign-investment